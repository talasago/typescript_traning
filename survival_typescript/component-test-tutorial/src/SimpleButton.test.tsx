import { render, screen } from "@testing-library/react";
import { SimpleButton } from "./SimpleButton";

test("The display of ON/OFF change with click the button", () => {
  render(<SimpleButton/ >);
  const simpleButton = screen.getByRole("button");
  expect(simpleButton).toHaveTextContent("OFF");
});
