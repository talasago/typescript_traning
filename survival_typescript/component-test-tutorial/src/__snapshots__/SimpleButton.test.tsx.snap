// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The display is OFF immediately after drawing 1`] = `
<div>
  <button>
    OFF
  </button>
</div>
`;
